### PR TITLE
Working confusion matrix code for int arrays

### PR DESCRIPTION
**Pull request by kumare3**

### CODE REVIEW

File: metricsproject/Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
```
Comment (EngHabu) [inline metricsproject/Untitled.ipynb]: A better name than Untitled?

File: metricsproject/Untitled.ipynb [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+   "execution_count": 58,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "@inputs(y_true=[Types.Integer], y_pred=[Types.Integer], title=Types.String, normalize=Types.Boolean, classes=[Types.String])\n",
+    "@outputs(matrix=[[Types.Integer]], visual=Types.Blob)\n",
+    "@python_task\n",
+    "def confusion_matrix(wf_params, y_true, y_pred, title, normalize, classes, matrix, visual):\n",
```
Comment (EngHabu) [inline metricsproject/Untitled.ipynb]: Can you add comments throughout the notebook?

File: metricsproject/Untitled.ipynb [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+   "execution_count": 42,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "FileNotFoundError",
+     "evalue": "[Errno 2] No such file or directory: '/var/folders/2b/wytvyh952jz8c17mp6yvh8nc0000gq/T/test_inbo8f8i/visual'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
```
Comment (EngHabu) [inline metricsproject/Untitled.ipynb]: Is showing an error here intended?

**Review by EngHabu — APPROVED** [review]

**Comment by bnsblue** [comment]
image
Should fix the figure before merging?

**Comment by bnsblue** [comment]
Also, what is the purpose of this last cell?
image

**Comment by bnsblue** [comment]
Merging

File: metricsproject/Untitled.ipynb (hunk 3 above)
Comment (kumare3) [reply metricsproject/Untitled.ipynb]: ohh i did not want to commit the notebook

**Pull request merged**